### ISSUE TITLE
[Technical Review] vol0_ch12 — RL value function not formalized; gradient direction misleading

### ISSUE THREAD

**Issue by dieterolson** [D-sorganization/AffineDrift#2536]
## File
`articles/The_Geometry_of_Motion/quarto/vol0_ch12_machine_learning.qmd`

### 1. Value function defined informally (lines 73–80)

The chapter states: *"RL algorithms seek to learn the **Value Function** V(state), which predicts the total sum of all future rewards..."*

This is imprecise — without a discount factor, infinite-horizon sums diverge. A correct ML chapter needs the standard MDP formalism:

```
MDP = (S, A, P, R, γ) with γ ∈ (0, 1).
State value under policy π:  V^π(s) = E_π [ Σ_{t=0}^∞ γ^t R_t | s_0 = s ]
Action value:                Q^π(s, a) = E_π [ Σ_{t=0}^∞ γ^t R_t | s_0 = s, a_0 = a ]
Optimal value:               V*(s) = max_π V^π(s).
Bellman equation:            V^π(s) = E[ R(s, π(s)) + γ V^π(s') ].
```

### 2. Gradient direction wrong in words (line 64)

Line 64: *"The Gradient (∇_W ℒ) simply points uphill towards higher error."*
Technically true (gradient points uphill), but combined with no statement that SGD steps in the **negative** gradient direction, a reader could be misled. Fix: *"Gradient descent steps in the direction `−∇_W ℒ`, moving weights downhill toward lower loss."*

### 3. Universal approximation claim unqualified (lines 30–33)

*"Neural networks function as universal function approximators"* — true but needs a caveat:
- Cybenko's theorem applies to infinitely wide single-layer networks.
- Approximation in principle ≠ learnable in practice.
- No convergence guarantee for gradient descent on non-convex objectives.

### 4. Policy-gradient methods not mentioned

Chapter only covers value-based RL. Modern practical RL (PPO, SAC, TRPO) is policy-gradient or actor-critic. Add one paragraph distinguishing:
- Value-based: learn `V` or `Q`, act greedy.
- Policy-gradient: directly optimize `π(a|s; θ)` via `∇_θ J(π) = E[∇_θ log π(a|s) · Q(s, a)]`.
- Actor-critic: both.

### 5. Loss function examples not concrete

Line ~70 is abstract. Make it concrete:
```
Supervised imitation:   ℒ = || π_θ(s) − a_expert ||²        (MSE to expert action)
Q-learning:             ℒ = ( r + γ max_a' Q(s', a'; θ⁻) − Q(s, a; θ) )²     (TD target)
Policy gradient:        ℒ = −E[ log π_θ(a|s) · Â(s, a) ]
```

### 6. Practical stability of training not addressed

Real-world RL requires experience replay, target networks, entropy regularization, exploration strategies (ε-greedy, curiosity). None mentioned.

## Required Fix

1. Replace the informal value-function prose with a formal MDP + Bellman definition as shown above.

2. Fix line 64's gradient-direction statement.

3. Add a one-paragraph "Limitations of universal approximation" box.

4. Add a subsection "Families of RL algorithms" covering value-based, policy-gradient, actor-critic.

5. Add the three concrete loss-function examples.

6. Add a short "Practical training concerns" subsection listing: exploration/exploitation tradeoff, sample efficiency (RL is data-hungry), sim-to-real gap, non-stationarity issues, target networks for stability.

7. Cite Sutton & Barto 2018 "Reinforcement Learning: An Introduction" as the canonical reference.

## Verification

- After fix, every RL concept used in the chapter is either defined in-chapter or explicitly deferred ("see Sutton & Barto §X"), with zero forward-usage of undefined symbols.
- The Bellman equation should typecheck dimensionally.

## Context

Found during website-wide technical accuracy audit #2530.

**Comment by dieterolson** [comment]
All seven required fixes are already present in staging (`vol0_ch12_machine_learning.qmd`): MDP formalism and Bellman equation (lines 91-104), explicit negative-gradient step (line 75), 'Caveats on Universal Approximation' callout (lines 35-45), 'Families of RL Algorithms' subsection (lines 106-114), concrete loss functions (lines 116-128), 'Practical Training Concerns' subsection (lines 130-140), and Sutton & Barto 2018 citation (line 140). Closing as resolved.
